src/dnc/azext_dnc/generated/_params.py

```diff
@@ -67,8 +61,8 @@ def load_arguments(self, _):
         c.argument('cluster_root_ca', type=str, help='RootCA certificate of kubernetes cluster base64 encoded')
         c.argument('api_server_endpoint', type=str, help='K8s APIServer url. Either one of apiServerEndpoint or '
                    'privateLinkResourceId can be specified')
-        c.argument('private_link_resource_id', type=str, help='private link arm resource id. Either one of '
-                   'apiServerEndpoint or privateLinkResourceId can be specified')
+        c.argument('private_link_resource_id', options_list=['--privlinkresourceid'], type=str, help='private link arm '
```

Comment:
Can you split it by ‘-‘

Reply:
I'm sorry I am not following, I currently have it like this - what split are we trying to accomplish? 

      - where:
            group: .*
            parameter: privateLinkResourceId
        alias:
            - privLinkResourceId

Reply:
Can you try the following configuration?

      - where:
            group: .*
            parameter: privateLinkResourceId
        alias:
            - priv_link_resource_id

Reply:
done